projects/batfish/src/org/batfish/grammar/cisco/Cisco_logging.g4

```diff
@@ -211,4 +214,24 @@ s_logging
       logging_common
       | logging_vrf
    )
-;
```

Comment:
I don't believe these should have been converted to rules. Were you just trying to refactor?